docs/core-components.md

```diff
@@ -15,6 +15,16 @@ An `Envelope` is the core object with which agents communicate. It is a vehicle
 * `message`: is a bytes field which holds the message in serialized form.
 
 
+### Skill
+
+Skills deliver economic value to the AEA by allowing agents to encapsulate and call any kind of code. They encapsulate the abstract base classes `Handler`, `Behaviour`, and `Task`.
+
+* `Handler`: each skill has a single `Handler` which is responsible for the registered protocol messaging. Handlers implement reactive behaviour. By understanding the requirements contained in an `Envelope`, the `Handler` reacts appropriately to message requests. 
```

Comment:
We are going to support multiple handlers per skill based on what @DavidMinarsch said.

Reply:
Correct, let's please update to multiple `Handler` objects